tools/inference.py

```diff
@@ -0,0 +1,162 @@
+import fire
+import numpy as np
+import pandas as pd
+import pathlib
+from penquins import Kowalski
+import tensorflow as tf
+from typing import List, Union
+import yaml
+
+
+config_path = pathlib.Path(__file__).parent.parent.absolute() / "config.yaml"
+with open(config_path) as config_yaml:
+    config = yaml.load(config_yaml, Loader=yaml.FullLoader)
+
+kowalski = Kowalski(
+    token=config["kowalski"]["token"],
+    protocol=config["kowalski"]["protocol"],
+    host=config["kowalski"]["host"],
+    port=config["kowalski"]["port"],
+)
+
+
+def get_features(
+    source_ids: List[int],
+    features_catalog: str = "ZTF_source_features_DR5",
+):
+    if not hasattr(source_ids, "__iter__"):
```

Comment:
Is this necessary given the List type checking above?

Reply:
Yes, bc it is used by `fire` in `__main__`, which is not smart enough to do the conversion on the fly when the user provides a single `source_id`.